packages/backend/src/main/java/com/codurance/open_space/OpenSpaceSessionController.java

```diff
@@ -26,6 +28,18 @@ public OpenSpaceSession create(@RequestBody OpenSpaceSession openSpaceSession) {
         return repository.save(openSpaceSession);
     }
 
+    @PutMapping("/{id}")
```

Comment:
You could also return `NO_CONTENT` from here also. The alternative is a 200 with the entity saved which I think you're returning here explicitly. Is that the default from Spring?